### DISCUSSION TITLE
What is Event Select and what does it do?

### DISCUSSION THREAD

**Discussion by Greminn** [tsightler/ring-mqtt#662] [Support Q&A] [answered]
Hi There, Just moved to ring-mqtt and it's fantastic! I understand all the controls and sensors, but the Event Select - could someone enlighten me what it does? I see a number of options under the select: Ding 1-5, Ding 1-5 (Transcoded), Motion 1-5 etc. Thanks :)

Note: Using Ring Wired Doorbell connected to Wifi via a Chime Pro v2.

<img width="335" alt="image" src="https://github.com/tsightler/ring-mqtt/assets/81659316/a73e1e65-1b5d-46ef-9cd8-7871abac9037">

**Comment by tsightler** [comment, marked as answer]
It is covered in the documentation for the event stream:
https://github.com/tsightler/ring-mqtt/wiki/Video-Streaming#event-stream

**Reply by Greminn** [reply]
Ah ok! Many thanks. I dont have a Ring Protect plan, so I guess I can safely disable this entity?

**Reply by tsightler** [reply]
Yes.  Theoretically the code should be detecting that you don't have a subscription and disabling these entities automatically, but there is a bug in 5.4.x versions right now where this is not working.
